index.ts

```diff
@@ -163,48 +189,52 @@ export function autocomplete ({
  * @param  {string} [$0.url]
  * @return {Promise}                A Promise that'll get resolved with the bulk geocode result
  */
-export function bulk ({
+export function bulk({
   addresses,
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   url
 }: BaseQuery & {
-  addresses: Array<any>,
-  boundary?: Boundary,
-  focusPoint?: any
+  addresses: Array<string>
+  boundary?: Boundary
+  focusPoint?: Point
+  text?: string
 }): Promise<BaseResponse> {
```

Comment:
The `BaseResponse` is not even correct here as the query part is an Object with the addresses key/value. Please update the return type.

Reply:
Fixed in cdd90839c90a85888732fb6718964b8ddd8fb1b6